Assets/Scripts/ObsticleController.cs

```diff
@@ -8,22 +8,32 @@ public class ObsticleController : MonoBehaviour
     float maxHeight;
     float maxWidth;
 
+    // give it random movement on (x) similar to (gust of wind)
+
     private void Awake()
     {
         RectTransform rt = FindObjectOfType<Canvas>().GetComponent<RectTransform>();
```

Comment:
Why not reference the [camera object](https://discussions.unity.com/t/spawn-object-at-random-points-within-camera-view/111240/2) instead?